tests/apollo/test_skvbc_reconfiguration.py

```diff
@@ -142,7 +142,7 @@ async def test_wedge_command_and_specific_replica_info(self, bft_network):
                 rsi_rep = client.get_rsi_replies()
                 done = True
                 for r in rsi_rep.values():
-                    if skvbc.parse_rsi_reply(rep, r) == 0:
+                    if skvbc.parse_rsi_reply(rep, r.get_rsi_data()) == 0:
```

Comment:
what's the reasoning behind this change?

Reply:
For singe message a reply message was returned using the last of the messages that reached quorum. Since we have parallel messages now for a single quorum I changed the RepliesManager to store the entire MsgWithReplicaSpecificInfo instead of just the rsi_data, so it can be used after quorum is reached. Looking at it now however I should change the get_rsi_replies method to still return the same as before instead of this.

Reply:
Thanks @KTeofanidis . Can you provide a bit more explanation about the changes in the `bft_client` ?